--- a/configure.ac
+++ b/configure.ac
@@ -13,7 +13,20 @@
 AC_SYS_LARGEFILE
 
 AC_CHECK_HEADERS([byteswap.h])
-AC_CHECK_FUNCS([nl_langinfo iconv])
+
+found_iconv=no
+# Check if we have GNU libiconv
+if test $found_iconv = "no"; then
+        AC_CHECK_LIB(iconv, libiconv_open, [with_libiconv=gnu; found_iconv=yes])
+fi
+# Check in the C library first
+if test $found_iconv = "no"; then
+        AC_CHECK_FUNC(iconv_open, [with_libiconv=no; found_iconv=yes])
+fi
+# Check if we have a iconv in -liconv, possibly from vendor
+if test $found_iconv = "no"; then
+        AC_CHECK_LIB(iconv, iconv_open, [with_libiconv=native; found_iconv=yes])
+fi
